apm-server/README.md

```diff
@@ -54,15 +52,6 @@ This chart is tested with the latest 7.9.2-SNAPSHOT version.
   - with [Helm 3 (beta)][]: `helm install apm-server elastic/apm-server`
```

Comment:
```suggestion
  - with [Helm 3 (beta)][]: `helm install apm-server --version 7.9.2 elastic/apm-server`
```

Reply:
All the installed instructions (Helm2 and Helm3 for all charts) should mention the `--version 7.9.2`